packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 46.14.0 - September 12, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Refactor FilePathExistsChecker for command usage in preparation of org browser multi-retrieve work ([PR #1629](https://github.com/forcedotcom/salesforcedx-vscode/pull/1629))
```

Comment:
I think we can remove this one. What do you think @brpowell ?

Reply:
Yeah this can be removed. It is a non-behavior changing check in.